### PR TITLE
[DOCS-371] Adding usecase of executing on certain days-of-week

### PR DESCRIPTION
**Pull request by rajkaramchedu**
This use case, execute on preferred days of week, does not introduce any new time-related API. Still the use case is valid to illustrate working with time. I'll think on coming up with a case that introduces one to two new time-related methods.

### CODE REVIEW

File: smartapp-developers-guide/time-methods.rst [hunk 1]
```diff
@@ -75,3 +78,51 @@ fromTime      toTime        new Date()   between
...
+A natural extension to the above automation of taking action within a time window is taking action only within a time window on *selected* days of the week.
+This can be easily achieved by a slight modification to the above SmartApp.
+
+First we prompt the user to select the preferred days of the week, by adding an enumerated ``input`` *Days* in the ``preferences`` section, as below: 
+
+.. code-block:: groovy
+
+    preferences {
+        section("On Which Days") {
+            input "Days", "enum", title: "Select Days of the Week", required: true, multiple: true, options: ["Monday": "Monday", "Tuesday": "Tuesday", "Wednesday": "Wednesday", "Thursday": "Thursday", "Friday": "Friday"]
```
Comment (jimmyjames) [inline smartapp-developers-guide/time-methods.rst]: this is super minor, but we should follow the standard java programming idiom of naming variables using camel case. In this case, it should just be all lowercase (`days`).

File: smartapp-developers-guide/time-methods.rst [hunk 2]
```diff
@@ -75,3 +78,51 @@ fromTime      toTime        new Date()   between
...
+
+    def contactHandler(evt) {
+
+        // Door is opened. Now check if today is one of the preset days-of-week
+        def df = new java.text.SimpleDateFormat("EEEE")
+        // Ensure the new date object is set to local time zone
+        df.setTimeZone(location.timeZone) 
+        def day = df.format(new Date())
+        //Does the preference input Days, i.e., days-of-week, contain today?
+        def dayCheck = Days.contains(day)
```
Comment (jimmyjames) [inline smartapp-developers-guide/time-methods.rst]: then just change `Days` here to `days`.

**Comment by rajkaramchedu** [comment]
Done.

**Comment by rajkaramchedu** [comment]
Fixed misplaced indentation.

**Comment by rajkaramchedu** [comment]
Also fixed *Days* to *days* in the text description.

**Comment by jimmyjames** [comment]
looks good 👍

**Pull request merged**